internal/security/common/tokenpolicy_test.go

```diff
@@ -13,7 +13,7 @@
 //
 // SPDX-License-Identifier: Apache-2.0'
 //
-package fileprovider
+package common
```

Comment:
update license header for year 2023

Reply:
Oh no!  Just pushed a commit to update copyright dates.